tests/test_gameboard.py

```diff
@@ -526,3 +526,27 @@ def _test_winner_in_diagonals(self, winner_method):
         B R R R . R B
         """
         self.assertEquals(getattr(board, winner_method)(), 'blue')
+
+    def test_is_valid_move_on_valid_column(self):
```

Comment:
Couple of things: 

- The tests and the ASCII charts are swapped (sample in 1 belongs to test 2 and viceversa).
- You are missing the test with the empty column.